packages/devextreme/js/ui/tabs.js

```diff
@@ -660,6 +678,9 @@ const Tabs = CollectionWidget.inherit({
             }
             case 'stylingMode': {
                 this._toggleStylingModeClass(args.value);
+                if(!this._isServerSide()) {
```

Comment:
[NOTE]
This's a dubious solution but we decided to refactor it in a separate card.